ratelimits/bucket.go

```diff
@@ -76,8 +76,9 @@ func newRegIdDomainBucketKey(name Name, regId int64, orderName string) (string,
 
 // newFQDNSetBucketKey validates and returns a bucketKey for limits that use the
 // 'enum:fqdnSet' bucket key format.
-func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) {
-	id := string(core.HashNames(orderNames))
+func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) { //nolint: unparam
```

Comment:
Why is the nolint directive necessary? Looks to me like all inputs and outputs are used.

Reply:
```
ratelimits/bucket.go:79:26: `newFQDNSetBucketKey` - `name` always receives `CertificatesPerFQDNSet` (`7`) (unparam)
func newFQDNSetBucketKey(name Name, orderNames []string) (string, error) {
```

Reply:
The message seems to be suggesting you could remove the first parameter and hard code `CertificatesPerFQDNSet` as the name inside this function. Sounds like a good idea to me; why not?

Reply:
I would prefer that these functions (`newFQDNSetBucketKey` and `newIPAddressBucketKey`) resemble their four siblings, or we simply inline their contents into their callers. Otherwise, I'd end up with two of these having the limit Name hard-coded and have to document that this be undone if we were to add another limit that uses the same identifier format for a bucket key (enum:identifier).